salt/modules/pillar.py

```diff
@@ -62,6 +63,24 @@ def get(key,
             then merging will be skipped and the retrieved value will be
             returned. Earlier releases raised an error in these cases.
 
+    deep_merge_lists : ``None``
```

Comment:
I considered `merge_lists` at first, but felt that the term could be confusing since this _only_ operates when the list is in a dictionary. If the list is not in a dictionary, then this code block is not entered.